package.json

```diff
@@ -48,6 +49,7 @@
   ],
   "license": "MPL-2.0",
   "dependencies": {
+    "@types/async": "^3.0.8",
     "async": "^2.6.1",
```

Comment:
We want at some point getting rid of the `async` dependencies - just as a side note - see issue here https://github.com/ethereumjs/organization/issues/55

Reply:
ah ok thanks

Reply:
Oops this should in devDeps, I’ll move it

Reply:
Oh, you are right. Already merged, just do a short follow-up PR eventually.

Reply:
No prob, will include in next PR for `promisify` work, hopefully today or tomorrow.